libraries/AP_HAL_ChibiOS/hwdef/AnyleafH7/defaults.parm

```diff
@@ -0,0 +1,25 @@
+# Onboard ELRS
+SERIAL5_PROTOCOL 23 # RCIN
+# RC_OPTIONS 512  # Suppress ELRS errors
+RC5_OPTION 154 # Set up arm switch to `ArmDisarm with AirMode`
+SERIAL5_BAUD 115
+RC2_REVERSED 1 # Prevent inverse pitch controls
+RSSI_TYPE 3 # Uart
+ARMING_RUDDER 0  # Disable rudder arming
+
+# Remove default serial protocols; set to None.
+SERIAL1_PROTOCOL -1
+SERIAL3_PROTOCOL -1
+SERIAL4_PROTOCOL -1
+
+
+# Onboard OSD
+OSD_TYPE 5 # MSP
+SERIAL2_PROTOCOL 42 # DisplayPort
+MSP_OPTIONS 4 # Betaflight fonts; without it, many symbols are missing.
```

Comment:
this will cause issues with Walksnail and DJI WTFOS....should leave this at default I think

Reply:
I can confirm that this is the correct setting for WTFOS, and O3.

Reply:
Also, I recognize that this is a subjective thing; for my own use case and the intended use case, it makes sense to set this default. For the general use case, it is better to leave these to the AP defaults. I think, based on the the fact that this FC has no analog OSD chip, and a DJI-format external port, it makes sense to have it set up by default for MSP Displayport OSD.

A relevant question, as you pointed out, is "Do HDZero and WalkSnail also use MSP DisplayPort (Like WTF OS and O3). I leave this to you or another AP maintainer to make the call; my vote is keep it set up for MSP Displayport like this.

Reply:
HD-Zero supports MSP-Displayport. For HD-Zero OSD_TYPE = 5, SERIALx_PROTOCOL = 42, but MSP_OPTIONS should not be set to 4 for HD-Zero. 
See the wiki https://ardupilot.org/plane/docs/common-msp-osd-overview-4.2.html#displayport-osd

Reply:
Removed MSP_OPTIONS set; great tip.